roles/requirements.yml

```diff
@@ -1,22 +1,6 @@
 ---
-##################################################
-# Requirements (Roles)
-##################################################
 
-collections: []
-
-roles:
-  - name: infra-ansible
-    scm: git
-
-    # TODO: Phase 1: Run from fork
-    src: https://github.com/MAHDTech/infra-ansible
-    version: feature/user-reset
-
-    # TODO: Phase 2: Run from main
-    #src: https://github.com/redhat-cop/infra-ansible
-    #version: main
-
-    # TODO: Phase 3: Run from release
-    #src: https://github.com/redhat-cop/infra-ansible
-    #version: v2.0.4
+- name: infra-ansible
```

Comment:
Let's look for a better way to maintain this dependency - e.g.: let's look for a way to use the infra-ansible project that is already set up in tower rather than having a dependency in here that may be though to remember/find when it needs to be updated.

Reply:
As discussed, leaving this for a seperate PR